x-pack/plugins/infra/common/http_api/infra_ml/results/metrics_hosts_anomalies.ts

```diff
@@ -62,6 +62,7 @@ export const getMetricsHostsAnomaliesRequestPayloadRT = rt.type({
     rt.type({
       // the ID of the source configuration
       sourceId: rt.string,
+      anomalyThreshold: rt.string,
```

Comment:
Wondering why this is a string and not a number?

Reply:
Was a bit torn here.  If i treated like a string it worked as a text input in the form easier as part of [FormState](https://github.com/elastic/kibana/pull/90313/files/06eb83151a1b49b3bf625320e7693cae046f3ae8#diff-a28c7639cca6418db123ce1729b0dd86a7aa93867fdd2f8f65961ff88ac16af7R27).  I could validate it as an empty string if the user left deleted the input and it could have a default value of '' like the other inputs.  I'll change it to a number as that seems most appropriate but it will default to 0 now if the user deletes the input.  Trying to convert the types made things messy.